src/HasMedia/HasMediaTrait.php

```diff
@@ -69,6 +69,28 @@ public function addMediaFromRequest(string $key)
     }
 
     /**
+     * Add multiple files from a request by keys.
+     *
+     * @param string[] $keys
+     *
+     * @return \Spatie\MediaLibrary\FileAdder\FileAdder[]
+     */
+    public function addMultipleMediaFromRequest(array $keys)
```

Comment:
Can't we make `addMediaFromRequest` variadic and allow it to accept arrays instead? Or not allowed for semver?